File: Practica9/Python/Ejercicio 2/Figura.py
from abc import ABC, abstractmethod

class Figura(ABC):
    def __init__(self, color):
        self.color = color

    def set_color(self, color):
        self.color = color

    def get_color(self):
        return self.color

    def __str__(self):
        return "Color: " + self.color

    @abstractmethod
    def area(self):
        pass

    @abstractmethod
    def perimetro(self):
        pass

File: Practica9/Java/Ejercicio 2/Figura.java
import java.awt.*;
abstract class Figura {
    String color;

    Figura(String c) {
        color = c;
    }

    String getColor() {
        return color;
    }

    abstract double area();
    abstract double perimetro();
    abstract void dibujar(Graphics g, int x, int y);
}